### PR TITLE
Fix Log Replication Failing IT

### PR DESCRIPTION
**Pull request by annym**
## Overview

Description:

Fix testLogEntrySyncValidCrossTablesWithWritingAtSrc IT

### CODE REVIEW

**Comment by pankti-m** [comment]
Is see a failure in testLogEntrySyncValidCrossTablesWithWritingAtSrc().  Is it a known issue which this is fixing?

**Comment by annym** [comment]
> Is see a failure in testLogEntrySyncValidCrossTablesWithWritingAtSrc(). Is it a known issue which this is fixing?

Yes this is the one I'm trying to fix. Let's wait for results to come back and I'll re-run a couple of times to confirm. Cause locally it does not fail any more.

File: test/src/test/java/org/corfudb/integration/LogReplicationIT.java [hunk 1]
```diff
@@ -1190,7 +1192,7 @@ private LogReplicationFSM startLogEntrySync(Set<String> tablesToReplicate, Set<W
         blockUntilExpectedAckTs.acquire();
         expectedAckTimestamp = srcDataRuntime.getAddressSpaceView().getLogTail();
 
-        // Block until the snapshot sync completes == one ACK is received by the source manager, or an error occurs
+        // Block until the expected ACK Timestamp is reached
         System.out.println("\n****** Wait until the wait condition is met");
         if (waitConditions.contains(WAIT.ON_ERROR)) {
```
Comment (xiaoqin2012) [inline test/src/test/java/org/corfudb/integration/LogReplicationIT.java:1197]: Change the system.out to log.in.

Comment (annym) [reply test/src/test/java/org/corfudb/integration/LogReplicationIT.java:1197]: System outs for testing are all over the place so even changing one does not do much. If this is something that we want we can make this in a separate PR where we change all of them.

**Pull request merged**